backend/test/api-test-example.test.js

```diff
@@ -0,0 +1,49 @@
+const supertest = require("supertest");
+const app = require("../app");
+const request = supertest(app);
+const dbHandler = require("./db-handler");
+
+const Event = require("../models/event.model.js");
+
+/**
+ * Connect to a new in-memory database before running any tests.
+ */
+beforeAll(async () => await dbHandler.connect());
+
+// Uncomment out the line below to clear out the DB of Users between tests.
+// afterEach(async () => await Event.remove({}));
+
+/**
+ * Remove and close the db and server.
+ */
+afterAll(async () => await dbHandler.closeDatabase());
+
+// API Tests
+describe("Test GET with no data in database", () => {
```

Comment:
Tests can have a describe section to describe the test with a string. See the output in the console when you run the tests.